itest/godot/ManualFfiTests.gd

```diff
@@ -326,3 +326,15 @@ func test_gd_self_reference_succeeds():
 	gd_self_reference.update_internal_signal.connect(update_self_reference)
 
 	assert_eq(gd_self_reference.succeed_at_updating_internal_value(10), 10)
+
+func sample_func():
+	pass
+
+func test_callable_refcount():
+	var test_obj: CallableRefcountTest = CallableRefcountTest.new()
+	for i in range(10):
+		var call: Callable = Callable(self, "sample_func")
```

Comment:
The name `call` is confusing, maybe `function` or `method` is better.

Also, you can use `:=` to avoid repeating the type.